.flake8

```diff
@@ -4,4 +4,4 @@ max-line-length = 79
 
 # E12x continuation line indentation
 
-ignore = E121
+ignore = E121,E722
```

Comment:
Can you put what's E722 about in the comment? So that we don't need to look up in the internet.

Reply:
I plan to fix it in the next few days :-) So this is only temporary